rero/rero-ils
# facets: expand facet items by link

* Adds a link to see more or less results on the facet.
* Closes #87

How to test:
- `script/bootstrap`
- Search in documents
- Check link on facets (see only if the facet results > 10)

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
